tests/disaster-recovery/regional-dr/test_failover_and_relocate.py

```diff
@@ -249,3 +332,49 @@ def test_failover_and_relocate(
             )
 
         # TODO: Add data integrity checks
+
+        # Get lastGroupSyncTime after relocate. The parameter lastGroupSyncTime may not be present in drpc yaml for
+        # some time after relocate. So the wait time given is more than the scheduling interval.
+        for drpc_data in TimeoutSampler(
+            (scheduling_interval * 60) + 300, 15, drpc_obj.get
+        ):
+            post_relocate_last_group_sync_time = drpc_data.get("status").get(
+                "lastGroupSyncTime"
+            )
+            if post_relocate_last_group_sync_time:
+                logger.info("After relocate - Obtained lastGroupSyncTime.")
+                # Adding an additional check to make sure that the old value is not populated again.
+                if (
+                    post_relocate_last_group_sync_time
+                    != post_failover_last_group_sync_time
+                ):
+                    logger.info(
+                        "After relocate - Verified: lastGroupSyncTime after relocate is different from the previous "
+                        "value."
+                    )
+                    break
+            logger.info(
+                "The value of lastGroupSyncTime in drpc is not updated after relocate. Retrying."
+            )
+        logger.info(
+            f"The value of lastGroupSyncTime after relocate is {post_relocate_last_group_sync_time}."
+        )
+
+        # Verify lastGroupSyncTime after relocate
+        time_format = "%Y-%m-%dT%H:%M:%SZ"
+        post_relocate_last_group_sync_time_formatted = datetime.strptime(
+            post_relocate_last_group_sync_time, time_format
+        )
+        current_time = datetime.strptime(
+            datetime.utcnow().strftime(time_format), time_format
+        )
+        time_since_last_sync = (
+            current_time - post_relocate_last_group_sync_time_formatted
+        ).total_seconds() / 60
+        logger.info(
+            f"After relocate - Time in minutes since the last sync is {time_since_last_sync}"
+        )
+        assert (
+            time_since_last_sync < 2.5 * scheduling_interval
```

Comment:
```suggestion
            time_since_last_sync < 2 * scheduling_interval
```

Reply:
Changed to 3 * scheduling_interval as discussed.